server/rpc/common.go

```diff
@@ -263,6 +271,7 @@ func buildGetBlockVerboseResult(s *Server, block *util.Block, isVerboseTx bool)
 		Bits:                 strconv.FormatInt(int64(blockHeader.Bits), 16),
 		Difficulty:           getDifficultyRatio(blockHeader.Bits, params),
 		ChildHashes:          childHashStrings,
+		AcceptedBlockHashes:  acceptedBlockHashesStrings,
```

Comment:
I think it's appropriate to call `daghash.Strings(acceptedBlockHashes)` here. Similar things are being done all-around, and it will save us the extra line and extra variable.